src/ape/api/transactions.py

```diff
@@ -27,8 +27,9 @@ class TransactionAPI(BaseInterfaceModel):
     """
 
     chain_id: int = Field(0, alias="chainId")
-    receiver: Optional[str] = Field(None, alias="to")
-    sender: Optional[str] = Field(None, alias="from")
+    hash: Any = Field(None, alias="txn_hash")
```

Comment:
We already have `txn_hash` computer property. We don't have a txn_hash property on a txn that is not pre-computed since you get the hash on the receipt